projects/bootloader/src/stm32f0xx/jump_to_bootloader.c

```diff
@@ -0,0 +1,30 @@
+#include "jump_to_bootloader.h"
```

Comment:
jump_to_application, jump_to_bootloader, and jump shouldn't be duplicated between the bootloader project and the ms-bootloader libraries, they should only appear in ms-bootloader. So remove the copies in the bootloader project / move all the functionality to ms-bootloader.